lms/static/sass/multicourse/_about_pages.scss

```diff
@@ -44,11 +44,11 @@
       padding-bottom: 40px;
 
       .logo {
-        border-right: 1px solid rgb(200,200,200);
+        @include border-right( 1px solid rgb(200,200,200) );
```

Comment:
@Bound3R To be consistent with the rest of our sass, could you please remove the extra spaces before and after parenthesis in this PR? So for example, line 47 should be

```
@include border-right(1px solid rgb(200,200,200));
````
and line 49 should be

````
@include float(left);
````